src/modules/75-cd/components/PipelineSteps/ShellScriptStep/shellScriptTypes.ts

```diff
@@ -72,3 +72,13 @@ export interface ShellScriptFormData extends StepElementConfig {
     source?: ShellScriptSource
   }
 }
+
+export enum ShellScriptSecrets {
```

Comment:
Can we rename this to a singular form? 
```suggestion
export enum ShellScriptSecret {
```